woocommerce/woocommerce-android
# Issue/12494 blaze webview fragment

Closes: #12494 and #12547


### Description

This PR adds 3 things: 
- Creates a new campaign detail webview to handle actions over the campaign. 
- Refreshes Blaze campaign list when coming back from campaign detail webview and the campaign has been stopped/cancelled
- Triggers Blaze native flow when user clicks on "Promote again" CTA

### Testing information


1. Log into a site with Blaze enabled
2. Create a Blaze campaign
3. From Blaze campaign list screen, click on the active campaign you've just created and Stop/Cancel it. 
4. Navigate back and check how the campaign list is updated with the most recent status updates
5. Click again on any campaign from the list. 
6. From the campaign detail webview click on "Promote again" CTA
7. See how the native campaign flow is triggered for the same product. 

### The tests that have been performed

- Tested Blaze campaign list refreshes correctly
- Tested promote again CTA triggers Blaze campaign creation. 

### Images/gif


https://github.com/user-attachments/assets/1d289bb8-c807-46d8-88f8-34f54b46a5f4


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
